### DISCUSSION TITLE
New Order doesn't make clear that shipping options may need to change

### DISCUSSION THREAD

**Discussion by bossanova808** [craftcms/commerce#2391] [Ideas]
**Description**

We have scenarios where the available shipping methods change depending on the order contents.  E.g. with just a calibrator in the cart, post options (standard/express) are offered.  But they then add something that is courier only, so now the only option is 'courier' perhaps. 

However, the invalid option remains selected and there is no visual indication it is no longer valid.  When you click on the shipping options, the old option is greyed out, but that's all the feedback you get really.

E.g. at this moment, courier is no longer valid as an option, but it's really not so clear.

![image](https://user-images.githubusercontent.com/731309/85092023-278bcd00-b22c-11ea-920d-66dae78d54aa.png)

Also, I am not so sure on the 'None' being a valid option/good presentation.  

I think it would be good to have some back end only options (like Free Shipping, for example) - for those instances where you're e.g. correcting a mistake with a previous order and can't charge shipping....but 'None' is not kind of very clear/explicit (and not having a handle on the order at all will probably break some of my templates as I am guessing I assume complete orders do have a method set on them....even if it is a $0 cost one!).

**Comment by lukeholder** [comment]
Looking in to adding a 'notices' list on an order for things that have changed during recalculation, in case the customer/editor needs to be aware of it. This could not only apply to a shipping method no longer being available, but also things like an item being removed from the cart due to it not longer being in stock or available.
